packages/angular/build/src/utils/i18n-options.ts

```diff
@@ -64,6 +65,16 @@ function ensureString(value: unknown, name: string): asserts value is string {
   }
 }
 
+function ensureValidateUrlSegment(value: unknown, name: string): asserts value is string {
```

Comment:
Style: `ensureValidate...` feels a bit redundant. Should this be `ensureValidUrlSegment` or `validateUrlSegment`?